torch/csrc/jit/mobile/interpreter.h

```diff
@@ -8,8 +8,10 @@ namespace torch {
 namespace jit {
 namespace mobile {
 using Stack = std::vector<c10::IValue>;
+using DebugHandle = int64_t;
 struct Code {
   std::vector<Instruction> instructions_;
+  std::vector<DebugHandle> debug_handles_;
```

Comment:
Yes, the main reason they are separate is due to the optional nature. I think it is fine to couple them. Let me do that in a later PR. Will put a TODO there.

Reply:
Sounds good.
As I said, I think right now they're pretty much coupled (i.e. debug_handles-.size() == instructions_.size() since we always add a -1 for when handles are not present).